rasa/nlu/tokenizers/spacy_tokenizer.py

```diff
@@ -47,17 +47,16 @@ def train(
                         MESSAGE_TOKENS_NAMES[attribute], self.tokenize(attribute_doc)
                     )
 
-    def get_doc(self, message, attribute):
-
+    def get_doc(self, message: Message, attribute: Text) -> "Doc":
         return message.get(MESSAGE_SPACY_FEATURES_NAMES[attribute])
 
     def process(self, message: Message, **kwargs: Any) -> None:
-
         message.set(
             MESSAGE_TOKENS_NAMES[MESSAGE_TEXT_ATTRIBUTE],
             self.tokenize(self.get_doc(message, MESSAGE_TEXT_ATTRIBUTE)),
         )
 
-    def tokenize(self, doc: "Doc") -> typing.List[Token]:
-
-        return [Token(t.text, t.idx) for t in doc]
+    def tokenize(self, doc: "Doc") -> List[Token]:
+        tokens = [Token(t.text, t.idx) for t in doc]
+        self.add_cls_token(tokens)
```

Comment:
technically there is still attribute here, we just exclude intent before hand

Reply:
yes, that is why I did not pass it forward to the method. do you think that is unclear?

Reply:
yes, because, by not passing, you use default value for `attribute`, which is `text`